lib/geocoder/lookups/db_ip_com.rb

```diff
@@ -0,0 +1,45 @@
+require 'geocoder/lookups/base'
+require 'geocoder/results/db_ip_com'
+
+module Geocoder::Lookup
+  class DbIpCom < Base
+
+    def name
+      'DB-IP.com'
+    end
+
+    def supported_protocols
+      [:http]
```

Comment:
Please add support here for https - since paid plans allow it. 
You may have to handle an error (in results method) when free plans set to https.